src/content/developers/docs/data-availability.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Data availability 
+description: An overview of problems and solutions relating to data availability in Ethereum 
+lang: en
+sidebar: true
+---
+
+In Ethereum, a node proposing a new block is required to make the underlying data available to the remaining peer-to-peer nodes on the network. This concept, known as data availability, is fundamental to the security and functionality of every blockchain, including Ethereum. Data availability is so important that an entire field of research studying the so-called “data availability problem” exists. 
+
+## What is data availability? {#what-is-data-availability}
+
+“Data availability” is a guarantee that all transaction data for a block was published by the block proposer and is available to other network participants. As explained in “[Intro to Ethereum](/developers/docs/intro-to-ethereum/)”, Ethereum transactions are processed in [blocks](/developers/docs/blocks/) and chained together to form the “blockchain”.
+
+Each block has two major parts: 
+
+- The **block header**: This contains general information (metadata) about the block, such as the timestamp, block hash, block number, etc.
+- The  **block body**: This contains the actual transactions processed as part of the block. 
+
+[Miner nodes](/developers/docs/consensus-mechanisms/pow/mining/) (or validators in [proof-of-stake](/developers/docs/consensus-mechanisms/pos/)) proposing new blocks must publish both the block header and block body. This allows nodes participating in [consensus](/developers/docs/consensus-mechanisms/) (full nodes) to download the block and confirm that transactions follow Ethereum’s consensus protocol. 
+
+[Full nodes](/developers/docs/nodes-and-clients/#full-node) are different from [light nodes](/developers/docs/nodes-and-clients/#light-node) that only download the block header. Without full nodes verifying transactions, block proposers could get away with inserting malicious transactions in blocks. 
+
```

Comment:
This section kind of makes it seem like miners are not also full nodes. Besides, referring to miners puts a short expiry date ont he page since block production will be done by validators after the merge. Suggest we refer to "block producers" instead of miners/validators? This section could be simplified by replacing with something like:

"All nodes execute the transactions in blocks they receive from peers to ensure that the changes proposed by the block producer precisely match those computed independently by the node. This is how nodes verify that new information is valid, rather than having to trust that block producers are honest. This is not possible if any data is missing".

Reply:
I used your suggestion as the basis for the new intro, since it's the ELI5 explanation @minimalsm wanted to see in the intro. I did adjust the original text in this section to a simpler variant based on your suggestion.